core/src/views.py

```
@@ -0,0 +1,355 @@
+class ValidationHistoryViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    ViewSet for ValidationHistory (read-only).
+
+    Endpoints:
+    - GET /api/validation-history/ - List validation history
+    - GET /api/validation-history/{id}/ - Retrieve specific validation
+    """
+
+    queryset = ValidationHistory.objects.select_related('article').all()
+    serializer_class = ValidationHistorySerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)